core/src/test/java/brooklyn/entity/drivers/ReflectiveEntityDriverFactoryTest.java

```diff
@@ -54,12 +58,22 @@ protected void assertDriverIs(Class<?> clazz) {
         assertTrue(driver.getClass().equals(clazz), "driver="+driver+"; should be "+clazz);
     }
     
+    protected void assertDriverIs(Class<?> clazz, Location location) {
```

Comment:
why not eliminate duplicate code and use this method for the existing test, too?